go-runtime/compile/testdata/pubsub/pubsub.go

```diff
@@ -0,0 +1,54 @@
+package pubsub
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/TBD54566975/ftl/go-runtime/ftl" // Import the FTL SDK.
+)
+
+type PayinEvent struct {
+	Name string
+}
+
+//ftl:verb
+func Payin(ctx context.Context) error {
+	if err := payinsVar.Publish(PayinEvent{Name: "Test"}); err != nil {
+		return fmt.Errorf("failed to publish event: %w", err)
+	}
+	return nil
+}
+
+//ftl:subscribe paymentProcessing
+func ProcessPayin(ctx context.Context, event PayinEvent) error {
+	logger := ftl.LoggerFromContext(ctx)
+	logger.Infof("Received PubSub event: %v", event)
+	return nil
+}
+
+var _ = ftl.RegisterSubscription(payinsVar, "paymentProcessing")
+
+var payinsVar = ftl.RegisterTopic[PayinEvent]("payins")
+
+var _ = ftl.RegisterSubscription(broadcast, "broadcastSubscription")
+
+// publicBroadcast is a topic that broadcasts payin events to the public.
+// out of order with subscription registration to test ordering doesn't matter.
+//
+//ftl:export
+var broadcast = ftl.RegisterTopic[PayinEvent]("publicBroadcast")
```

Comment:
Maybe we should require that the variable is also exported so that it matches the generated code.

Reply:
I wonder the same thing. It still won't match unless they name it the same though. Like in the above example, `PublicBroadcast` vs `Broadcast`